example/src/main/res/values/strings.xml

```diff
@@ -1,4 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
     <string name="app_name">RxExtensions</string>
+    <string name="post_activity_title">Create Post</string>
```

Comment:
please prefix strings

create_post_title and create_post_accept_button_content_description